### DISCUSSION TITLE
Documentation updates re. Windows kernel driver options?

### DISCUSSION THREAD

**Discussion by njk42** [libusb/libusb#1489] [Q&A]
I'm not reporting an issue here, rather hoping for clarification on Windows kernel driver options as discussed in the libusb FAQ/wiki.  (I hope this is an appropriate place to ask this question because I know there are several contributors who are involved across several of these USB driver & library projects and know their ins-and-outs very well.)

Currently and historically, libusb-win32 (libusb0.sys) has been listed as "not-recommended"; instead recommending libusbk.sys or WinUSB.sys.  However, I have noticed that libusb-win32 project has recent activity and updates (release v1.4.0).  Is it still the case that libusb-win32 is not recommended/preferred, or has this recent activity in libusb-win32 project made the libusb FAQ/wiki outdated?

**Comment by mcuee** [comment]
Maybe the question is what is the reason you want to use libusb-win32 device driver?

The issue with supporting libusb-win32 device driver under libusb Windows have been mostly sorted out. On the other hand, I will still rank the support level as WinUSB > libusbK > libusb0.sys.

 Take note the filter driver is not working for some devices and there are still issues supporting the filter driver in libusb Windows.

**Reply by mcuee** [reply]
@njk42 

I have updated the following wiki pages. Please take a look to see if you have any further comments. Thanks.
https://github.com/libusb/libusb/wiki/FAQ
https://github.com/libusb/libusb/wiki/Windows
